### ISSUE TITLE
Sandbox CRS version option results in empty reply

### ISSUE THREAD

**Issue by markhobson** [coreruleset/coreruleset#4308]
### Describe the bug

When specifying the HTTP header `x-crs-version` the request is terminated.

### Steps to reproduce

Using the [advanced example](https://coreruleset.org/docs/6-development/6-4-using-the-crs-sandbox/#advanced-examples) in the docs:

```bash
curl -H "x-backend: nginx" \
  -H "x-crs-version: 3.2.1" \
  -H "x-format-output: json-matched-rules" \
  https://sandbox.coreruleset.org/?file=/etc/passwd | jq .
```

### Expected behaviour

A JSON response as per the docs.

### Actual behaviour

An empty reply error message:

```
$ curl -H "x-backend: nginx" \
  -H "x-crs-version: 3.2.1" \
  -H "x-format-output: json-matched-rules" \
  https://sandbox.coreruleset.org/?file=/etc/passwd | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (52) Empty reply from server
```

### Additional context

Removing the CRS version header works normally:

```
$ curl -H "x-backend: nginx" \
  -H "x-format-output: json-matched-rules" \
  https://sandbox.coreruleset.org/?file=/etc/passwd | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   268    0   268    0     0    101      0 --:--:--  0:00:02 --:--:--   101
[
  {
    "id": "930120",
    "message": "OS File Access Attempt",
    "paranoia_level": "1"
  },
  {
    "id": "932160",
    "message": "Remote Command Execution: Unix Shell Code Found",
    "paranoia_level": "1"
  },
  {
    "id": "949110",
    "message": "Inbound Anomaly Score Exceeded (Total Score: 10)",
    "paranoia_level": "1"
  }
]
```

### Your Environment

* CRS version (e.g., v3.2.0): 3.2.1
* Paranoia level setting: sandbox default (PL1)
* ModSecurity version (e.g., 2.9.3): sandbox
* Web Server and version (e.g., apache 2.4.41): sandbox (nginx-latest)
* Operating System and version: Ubuntu 24.04.3 LTS

**Comment by fzipi** [comment]
Hi @markhobson! The only versions supported are latest 4.3.z and 4.y.z

**Comment by markhobson** [comment]
Hi @fzipi, ah right thanks! The [docs](https://coreruleset.org/docs/6-development/6-4-using-the-crs-sandbox/#default-options) say that the default version is 3.3.2 which misled me. Are there any plans to update them?

**Comment by markhobson** [comment]
Also shame to drop 3.3.x support as that's still used by GCP [Cloud Armor](https://cloud.google.com/armor/docs/waf-rules).

**Comment by fzipi** [comment]
Wait, we didn't drop 3.3.x support 😄
